client/order/epochqueue.go

```diff
@@ -0,0 +1,165 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package order
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"sort"
+	"sync"
+
+	"decred.org/dcrdex/dex/msgjson"
+	"decred.org/dcrdex/dex/order"
+	"github.com/decred/dcrd/crypto/blake256"
+)
+
+// EpochQueue represents a client epoch queue.
+type EpochQueue struct {
+	orders map[order.OrderID]order.Commitment
```

Comment:
`Commitment` is all that's needed right now, but we will need order type, side, and value when implemented. See https://github.com/decred/dcrdex/blob/f981df6e9905bb7aa68a008cb87c9a9a1a98be6d/client/core/types.go#L131-L141.